src/GPL-2.0.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <SPDXLicenseCollection xmlns="http://www.spdx.org/license">
-   <license isOsiApproved="true" licenseId="GPL-2.0"
+   <license isOsiApproved="true" isFsfLibre="true" licenseId="GPL-2.0"
```

Comment:
As I've [said][1] [earlier][2], I'd much rather be pulling this from an FSF API.  In the absence of such an API (the current situation) [I][3] [think][4] we should put this information in a mock FSF API which we can hand over to the FSF when they're ready to step up and maintain it.  I really don't like baking other people's metadata into our root data repository.  I would much rather this information be included in [license-list-data][5] after being pulled from the (mock) FSF API, instead of including the metadata here.

I am ok baking in an FSF license ID (if they decide to diverge from SPDX IDs in their API), at least until we address #418.

[1]: https://lists.spdx.org/pipermail/spdx-legal/2017-October/002282.html
[2]: https://github.com/spdx/spdx-spec/issues/46#issuecomment-336524889
[3]: https://lists.spdx.org/pipermail/spdx-legal/2017-October/002287.html
[4]: https://github.com/spdx/spdx-spec/issues/46#issuecomment-336576296
[5]: https://github.com/spdx/license-list-data

Reply:
Agree in principle, but I don't want to screen scrape for the information.   We could have a mock API that would work from a config file.  It should present the data through a web API, preferably REST based

Reply:
Agree in principle, but I don't want to screen scrape for the information.   We could have a mock API that would work from a config file.  It should present the data through a web API, preferably REST based

Reply:
I was imaginong we dump some JSON into GitHub.  I'll sketch that out tonight.

Reply:
JSON works.  If you put together sending side - I can update the SPDX license generator to use it.
@kestewart - Can you just put together a list of the licenses you think are FSF Libre?

Reply:
I'll pull the list using JavaScript, as I showed on the mailing list.

Reply:
@goneall - sure.   Just a flat text file of one SPDX-license-identifier per line for the subset that are isFsfLibre?